index.html

```diff
@@ -351,5 +354,6 @@ <h3 class="best">
         </ul>
       </div>
     </section>
+    <script src="main.js"></script>
   </body>
 </html>
```

Comment:
- [ ] **Optional:** Please add a screenshot of your project. https://github.com/Yoz21/Portfolio/blob/ab8a1a282b62b70bbe824f5bc6fb03299be56b25/README.md